.ci_support/migrations/python312.yaml

```diff
@@ -0,0 +1,38 @@
+migrator_ts: 1695046563
+__migrator:
+    migration_number: 1
+    operation: key_add
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.11.* *_cpython
+            - 3.12.* *_cpython  # new entry
```

Comment:
Hmm, still not running 3.12 in the pipeline. I say we merge this because I think this particular workflow file will create a new PR introducing all the `3.12` stuff.

cc: @agriyakhetarpal @priyanshuone6